boards.txt

```diff
@@ -34919,6 +34919,184 @@ XIAO_ESP32S3.menu.EraseFlash.all.upload.erase_cmd=-e
+XIAO_ESP32S3_PLUS.menu.FlashMode.dio.build.flash_mode=dio
+XIAO_ESP32S3_PLUS.menu.FlashMode.dio.build.boot=dio
+XIAO_ESP32S3_PLUS.menu.FlashMode.dio.build.boot_freq=80m
+XIAO_ESP32S3_PLUS.menu.FlashMode.dio.build.flash_freq=80m
+
+XIAO_ESP32S3_PLUS.menu.FlashSize.8M=8MB (64Mb)
+XIAO_ESP32S3_PLUS.menu.FlashSize.8M.build.flash_size=8MB
+
+XIAO_ESP32S3_PLUS.menu.LoopCore.1=Core 1
+XIAO_ESP32S3_PLUS.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+XIAO_ESP32S3_PLUS.menu.LoopCore.0=Core 0
+XIAO_ESP32S3_PLUS.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+XIAO_ESP32S3_PLUS.menu.EventsCore.1=Core 1
+XIAO_ESP32S3_PLUS.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+XIAO_ESP32S3_PLUS.menu.EventsCore.0=Core 0
+XIAO_ESP32S3_PLUS.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+XIAO_ESP32S3_PLUS.menu.USBMode.hwcdc=Hardware CDC and JTAG
+XIAO_ESP32S3_PLUS.menu.USBMode.hwcdc.build.usb_mode=1
+XIAO_ESP32S3_PLUS.menu.USBMode.default=USB-OTG (TinyUSB)
+XIAO_ESP32S3_PLUS.menu.USBMode.default.build.usb_mode=0
+
+XIAO_ESP32S3_PLUS.menu.CDCOnBoot.default=Enabled
+XIAO_ESP32S3_PLUS.menu.CDCOnBoot.default.build.cdc_on_boot=1
+XIAO_ESP32S3_PLUS.menu.CDCOnBoot.cdc=Disabled
+XIAO_ESP32S3_PLUS.menu.CDCOnBoot.cdc.build.cdc_on_boot=0
+
+XIAO_ESP32S3_PLUS.menu.MSCOnBoot.default=Disabled
+XIAO_ESP32S3_PLUS.menu.MSCOnBoot.default.build.msc_on_boot=0
+XIAO_ESP32S3_PLUS.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+XIAO_ESP32S3_PLUS.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+XIAO_ESP32S3_PLUS.menu.DFUOnBoot.default=Disabled
+XIAO_ESP32S3_PLUS.menu.DFUOnBoot.default.build.dfu_on_boot=0
+XIAO_ESP32S3_PLUS.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+XIAO_ESP32S3_PLUS.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+XIAO_ESP32S3_PLUS.menu.UploadMode.default=UART0 / Hardware CDC
+XIAO_ESP32S3_PLUS.menu.UploadMode.default.upload.use_1200bps_touch=false
+XIAO_ESP32S3_PLUS.menu.UploadMode.default.upload.wait_for_upload_port=false
+XIAO_ESP32S3_PLUS.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+XIAO_ESP32S3_PLUS.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+XIAO_ESP32S3_PLUS.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.default_8MB=Default with spiffs (3MB APP/1.5MB SPIFFS)
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.max_app_8MB=Maximum APP (7.9MB APP No OTA/No FS)
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.max_app_8MB.build.partitions=max_app_8MB
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.max_app_8MB.upload.maximum_size=8257536
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.tinyuf2=TinyUF2 8MB (2MB APP/3.7MB FFAT)
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.tinyuf2.build.custom_bootloader=bootloader-tinyuf2
+XIAO_ESP32S3_PLUS.menu.PartitionScheme.tinyuf2.build.custom_partitions=partitions-8MB-tinyuf2
```

Comment:
The partition name does not match with the partition file name you added to the variant folder.
`partitions-8MB-tinyuf2` vs `partitions-8MB`. Please fix the naming